Frameworks/UIKit/UIButton.mm

```diff
@@ -243,6 +279,7 @@ - (void)_initUIButton {
 - (instancetype)initWithFrame:(CGRect)frame {
     if (self = [super initWithFrame:frame]) {
         [self _initUIButton];
+        self.font = [UIFont buttonFont];
```

Comment:
because that would result in us always setting the font twice for xib-loaded UIButtons (initWithNib starts with a call to _initUIButton).

---
In reply to: [107235119](https://github.com/Microsoft/WinObjC/pull/2269#discussion_r107235119) [](ancestors = 107235119)